src/test/java/teammates/test/cases/webapi/UpdateStudentActionTest.java

```diff
@@ -177,6 +177,29 @@ public void testExecute() throws Exception {
                 invalidParamsOutput.getMessage());
     }
 
+    @Test
+    public void testExecute_withTeamNameAlreadyExistsInAnotherSection_shouldFail() {
```

Comment:
You should add a test that exactly test the scenario? `Should not be able to exceed section limit by editing section name of an existing student`

Reply:
Updated with new test  case